docs/getting-started/email-setup.md

```diff
@@ -0,0 +1,12 @@
+# Email Setup
+
+Logging emails is a central activity to Hubspot and allows for greater transparency and provides an opportunity for team members to have a deeper understanding of where a relationship is at. When configured Hubspot email logging allows to set up automated and/or selective logging of emails while you are in your prefered email tool
+
+Hubspot email logging can happen in two ways, you can either set up your email within Hubspot, or you install the [Hubspot email extension](https://knowledge.hubspot.com/connected-email/how-to-install-hubspot-sales). More details on logging will be found in the Activities section.
```

Comment:
> Hubspot email logging can happen in two ways, you can either set up your email within Hubspot, or you install the [Hubspot email extension]. More details on logging will be found in the Activities section.

Edit: Hubspot email logging can happen one of two ways; you can either set up your email within Hubspot, or you can install the [Hubspot email extension]. More details on logging can be found in the Activities section.

**Will "Activities Section" be a link once that section is added?

Reply:
Yeah, the activities link will be added once it is added online.